specification/monitor/resource-manager/readme.md

```diff
@@ -27,16 +27,27 @@ These are the global settings for the MonitorClient API.
 ``` yaml !$(python) || !$(track2)
 title: MonitorClient
 ```
+
 ``` yaml $(python) && $(track2)
 title: MonitorManagementClient
 ```
 
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2020-03
+tag: package-preview-2017-09
 ```
 
+
+### Tag: package-preview-2017-09
+
+These settings apply only when `--tag=package-preview-2017-09` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2017-09'
```

Comment:
Please add -only suffix for your tag because it only contains a part of monitor Swagger files.
```suggestion
### Tag: package-2017-09-preview-only

These settings apply only when `--tag=package-2017-09-preview-only` is specified on the command line.

```yaml $(tag) == 'package-2017-09-preview-only'
```

Reply:
Thanks for reviewing this and catching this, I've incorporated your feedback.